pkgs/development/tools/mysql-shell/innovation.nix

```diff
@@ -114,6 +114,10 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optionals stdenv.hostPlatform.isLinux [ libtirpc ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [ darwin.libutil ];
 
+  env = {
+    ${if stdenv.cc.isGNU then "NIX_CFLAGS_COMPILE" else null} = "-Wno-error";
```

Comment:
Is this not an eval failure on other platforms?

Reply:
Clang on Darwin is fine. I tried downgrading GCC, but it didn’t work. I’m not familiar with C++, so I didn’t try adjusting the code.

Reply:
``-Wno-error`` is a developer flag. It should not be set in prod builds as every little warning fails the build and with new gcc updates there are sometimes new ones and it is an incredible chore to fix those all on a distro level.